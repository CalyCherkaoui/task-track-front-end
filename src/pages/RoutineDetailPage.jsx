import React from 'react';
import { useParams } from 'react-router-dom';

const RoutineDetailPage = () => {
  const { routineid } = useParams();
  return (
    <div>
      {routineid}
    </div>
  );
};

export default RoutineDetailPage;
