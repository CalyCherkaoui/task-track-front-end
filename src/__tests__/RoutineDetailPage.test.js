import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../store';
import RoutineDetailPage from '../pages/RoutineDetailPage';

describe(
  'Renders correctly the RoutineDetail page',
  () => {
    it(
      'Matches RoutineDetailPage snapshot',
      () => {
        const rendered = renderer.create(
          <Provider store={store}>
            <Router>
              <RoutineDetailPage />
            </Router>
          </Provider>,
        );

        expect(rendered).toMatchSnapshot();
      },
    );
  },
);
