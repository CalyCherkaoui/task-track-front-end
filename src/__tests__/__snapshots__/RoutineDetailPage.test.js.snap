// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly the RoutineDetail page Matches RoutineDetailPage snapshot 1`] = `null`;
